packages/aws-cdk-lib/aws-stepfunctions/lib/states/state.ts

```diff
@@ -438,6 +452,40 @@ export abstract class State extends Construct implements IChainable {
     });
   }
 
+  /**
+   * Render ItemProcessor in ASL JSON format
+   */
+  protected renderItemProcessor(): any {
+    if (!this.processor) return undefined;
+    return {
+      ItemProcessor: {
+        ...this.renderProcessorConfig(),
+        ...this.processor?.toGraphJson(),
+      },
+    };
+  }
+
+  /**
+   * Render ProcessorConfig in ASL JSON format
+   */
+  private renderProcessorConfig(): any {
```

Comment:
```suggestion
  private renderProcessorConfig() {
```

`any` doesn't help much, i'd rather TS guses at the return type

Reply:
Makes sense, I specified the return type as this is the approach used for the other rendering functions.

Reply:
Fair :). If there's a reason for it I can't think of one, but it's whatever either way.